config/bags.lua

```diff
@@ -98,6 +98,8 @@ function config:GetBagOptions(kind)
             values = {
               ["RecentItems"] = L:G("Recent Items"),
               ["Type"] = L:G("Type"),
+              ["Soulbound"] = L:G("Soulbound"),
+              ["Unbound"] = L:G("Unbound"),
```

Comment:
See comment below, but can we come up with a better label for Unbound? Unbound can apply to anything that isn't soulbound.

Reply:
See below. I think its worth distingushing 'never binding' items from pre-bound items.